### PR TITLE
qt: General CSS related redesigns

### PR DESCRIPTION
**Pull request by xdustinface**
This PR ist part of a series of +-25 PRs related to UI redesigns. Its ancestor is #3562, its successor is  #3564. I did not screenshot every single PR and its changes, instead i made "walk through all screen" videos with the result of this PR series and also with the 0.15 UI. If there are any concrete screenshots wanted, just let me know. To build with the full set of changes you can build from the branch [xdustinface:pr-ui-redesign](https://github.com/xdustinface/dash/tree/pr-ui-redesign) which always contains all changes.

[ -> Walk through 0.15](https://youtu.be/cWQeHj5tWR0)
[ -> Walk through Redesign](https://youtu.be/0QeSyXo1aao)

I tried to give the commits enough text to make things obvious without a lot description for each PR. Also here, if you want more description for this specific PR, let me know.
### About this PR

See individual commit messages.

### CODE REVIEW

**Comment by xdustinface** [comment]
Ready for review! 

I combined all of #3563, #3564, #3565, #3566 into this one here. If thats too much let me know and i can split it again but i thought its all only css related... so why not :)

File: src/qt/res/css/general.css [hunk 1]
```diff
@@ -68,7 +68,7 @@ QDialog {
 }
 
 .QFrame {
-    background-color: transparent;
+    background-color: #00000000;
```
Comment (PastaPastaPasta) [inline src/qt/res/css/general.css:57]: can you explain the reason for this?

Comment (xdustinface) [reply src/qt/res/css/general.css:57]: Just that relevant colors are all in the same `#aarrggbb` format and that its catched by `contrib/devtools/update-css-files.py`. There is still the named color `red` but thats only a placeholder to show an undefined behaviour in the UI. So if there is a bright red somewhere in the UI it means the related color in `general.css` isn't overwritten in the theme file. I want to write some UI development style guidelines anytime later where i will explain such thoughts i had.

**Comment by UdjinM6** [comment]
Looks very nice in general 👍  traditional theme is a bit weird though and needs few more fixes imo, pls see 6639cb3e6e

**Comment by xdustinface** [comment]
thanks :) the thing with the traditional is weird.. will push the fix for sure but when i made the walkthrough video i hadn't this bugs in there and now i wonder where on the road i messed with that.. need to investigate here a little first if i messed something else too 🤔

**Comment by xdustinface** [comment]
Ok no, all-clear, nothing is lost you just got me confused haha.. its the same like back there i just didn't realise that i had the text color twice in there when looked at your diff earlier and then i thought the tab button texts are red but thats overwritten below and you just removed it 🙈 Picked your commit and updated the colors sections in the css files.

**Pull request merged**